packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -505,6 +463,18 @@ function countTreesInSnapshotTree(snapshotTree: ISnapshotTree): number {
     return numTrees;
 }
 
+/**
+ * This function fetches the snapshot and parse it according to what is mentioned in response headers.
+ * @param odspResolvedUrl - resolved odsp url.
+ * @param storageToken - token to do the auth for network request.
+ * @param snapshotOptions - Options used to specify how and what to fetch in the snapshot.
+ * @param logger - logger
+ * @param fetchBinarySnapshotFormat - whether to fetch binary snapshot or not.
+ * @param controller - abort controller if caller needs to abort the network call.
+ * @param epochTracker - epoch tracker used to add/validate epoch in the network call.
+ * @param disableTreesLatestForBinaryWireFormat - whether to disable trees/latest to fetch binary wire format
```

Comment:
I find this name misleading. I think easier to follow names would be 
- UseBinarySnapshotFormat / UseJsonSnapshotFormat
- UseV1SnapshotFormat / UseV2SnapshotFormat

Reply:
Or maybe "allow" is the verb that we should use?

Reply:
Removed this argument.